google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -532,13 +549,25 @@ public Boolean getIsLive() {
         return matchesStorageClass;
       }
 
+      /* Returns the date in RFC 3339 format with only the date part (for instance, "2013-01-15")*/
```

Comment:
End this with a period, i.e. "(for instance, "2013-01-15"). */"

Reply:
done